src/Services/QueuedJobService.php

```diff
@@ -1044,19 +1056,27 @@ protected function withNestedState(callable $callback)
         }
     }
 
+    protected function handleJobInitialisationException(QueuedJobDescriptor $jobDescriptor, Throwable $e)
```

Comment:
Why is this separate from `handleBrokenJobException()`?

Reply:
Because we don’t have a `$job` instance to pass to `handleBrokenJobException()`, and changing the method signature to make that argument optional is a BC break (plus it’d mean putting an optional argument ahead of a required argument, which is a bit yucky 😛)

Reply:
Couldn't you just pass in `null`?

Reply:
Nope, the argument is typehinted as `QueuedJob $job` so passing `null` will error

Reply:
Ahh yup fair enough.